etc/config/objc++.amazon.properties

```diff
@@ -842,6 +842,13 @@ libs.fastor.versions.trunk.path=/opt/compiler-explorer/libs/fastor/trunk
 libs.fastor.versions.063.version=0.6.3
 libs.fastor.versions.063.path=/opt/compiler-explorer/libs/fastor/V0.6.3/
 
+libs.flux.name=Flux
+libs.flux.description=A C++20 library for sequence-orientated programming. Safer, easier, faster, compatible.
+libs.flux.versions=trunk
+libs.flux.url=https://github.com/tcbrindle/flux
+libs.flux.versions.trunk.version=trunk
+libs.flux.versions.trunk.path=/opt/compiler-explorer/libs/flux/trunk/include
```

Comment:
With the install you sent for infra, this is being installed as:
```
$ ls /opt/compiler-explorer/libs/flux/
CMakeLists.txt  LICENSE_1_0.txt  README.md  benchmark  cmake  docs  example  include  module  single_include  test  tools
```

... I suspect that's a mistake so I'll fix that up and reinstall, then merge this. Otherwise this lgtm!